tests/python/tir-base/test_tir_specialize.py

```diff
@@ -65,7 +65,7 @@ def matmul_m_128(a: T.handle, b: T.handle, c: T.handle) -> None:
             C[vi, vj] = C[vi, vj] + A[vi, vk] * B[vj, vk]
 
 
-@T.prim_func
+@T.prim_func(check_well_formed=False)
```

Comment:
Not clear to me what's wrong here

Reply:
I believe this is from `x` being undefined in the TIR.  It appears as part of a shape in `T.match_buffer`, but as `x * 8*` rather than on its own.

Reply:
Hm, is this a legitimate test case then? I am not sure. It would be easy enough to add x as an argument, but would that still make sense as a test?